File: development/sphinx-console-coordinator/src/main/java/ru/skuptsov/sphinx/console/coordinator/model/SimpleCollectionWrapper.java
package ru.skuptsov.sphinx.console.coordinator.model;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class SimpleCollectionWrapper implements Serializable {
	private static final long serialVersionUID = 270647L;
	 
	private String collectionName;
	 
	private List<SimpleCollectionReplicaWrapper> agents = new ArrayList<SimpleCollectionReplicaWrapper>();

	public String getCollectionName() {
		return collectionName;
	}

	public void setCollectionName(String collectionName) {
		this.collectionName = collectionName;
	}

	public List<SimpleCollectionReplicaWrapper> getAgents() {
		return agents;
	}

	public void setAgents(List<SimpleCollectionReplicaWrapper> agents) {
		this.agents = agents;
	}
	 
	 
}

File: development/sphinx-console-web/src/main/webapp/sphinx-console/app/model/SimpleCollectionWrapper.js
Ext.define('sphinx-console.model.SimpleCollectionWrapper', {
    extend: 'Ext.data.Model',
    idProperty: 'collectionName',
    fields: [
        {name: 'collectionName', type: 'string'}
        ],
    hasMany:[
        {model: 'sphinx-console.model.SimpleCollectionReplicaWrapper', name: 'agents', associationKey: 'agents'}
    ]
});
